src/lib/mock-observations/helpers/rand-int-generator.ts

```diff
@@ -46,6 +46,14 @@ export const RandIntGenerator = (
         })
       );
     }
+
+    if (config.min >= config.max) {
+      throw new InputValidationError(
+        errorBuilder({
+          message: `Min value should not be greater than max value of ${validatedName}`,
```

Comment:
@manushak As we don't accept the equality between min and max, then let's rephrase the err. message to `Min value should not be greater than or equal to max value of ${validatedName}`